src/SimpleExcelReader.php

```diff
@@ -314,15 +325,17 @@ protected function getSheet(): SheetInterface
         $this->reader->open($this->path);
 
         foreach ($this->reader->getSheetIterator() as $key => $sheet) {
-            if ($key === $this->sheetNumber) {
+            if (($this->sheetNumber !== -1 && $key === $this->sheetNumber) || ($this->sheetName != "" && $this->sheetName === $sheet->getName())) {
                 break;
             }
         }
-
-        if ($this->sheetNumber !== $key) {
+        if ($this->sheetNumber !== -1 && $this->sheetNumber !== $key) {
             throw new InvalidArgumentException("Sheet {$key} does not exist in {$this->path}.");
+        } else {
```

Comment:
I don't think an else is needed as the positive branch throws an exception.

Reply:
My elle is because the is an other if insiste to verify the sheetName, it's not useless, i think, no ? Okay to the readme, I do this tomorrow.